script.js

```diff
@@ -14,4 +14,97 @@ close.addEventListener('click', () => {
+const popProject3 = document.getElementById('btn-3');
+const popProject = document.getElementById('btn-4');
+
+const projArr = [
+  {
+    id: 1,
+    pName: 'BENtech',
+    pAuthure: 'Benjamin &nbsp; &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2022',
+    pPic: 'photo/ben/ben4.png',
+    pText: 'An open source code, created and designed by Benjamin Ejike while studying with microverse, kindly report any kink of copyright.',
+    pTechs: ['Html', 'Css', 'JavaScript'],
+    pLive: 'https://osbentech.github.io/Portfolio/',
+    pSource: 'https://github.com/osbentech',
+  },
+  {
+    id: 2,
+    pName: 'BENtech',
+    pAuthure: 'Benjamin &nbsp; &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2022',
+    pPic: 'photo/ben/ben4.png',
+    pText: 'An open source code, created and designed by Benjamin Ejike while studying with microverse, kindly report any kink of copyright.',
+    pTechs: ['Html', 'Css', 'JavaScript'],
+    pLive: 'https://osbentech.github.io/Portfolio/',
+    pSource: 'https://github.com/osbentech',
+  },
+  {
+    id: 3,
+    pName: 'BENtech',
+    pAuthure: 'Benjamin &nbsp; &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2022',
+    pPic: 'photo/ben/ben4.png',
+    pText: 'An open source code, created and designed by Benjamin Ejike while studying with microverse, kindly report any kink of copyright.',
+    pTechs: ['Html', 'Css', 'JavaScript'],
+    pLive: 'https://osbentech.github.io/Portfolio/',
+    pSource: 'https://github.com/osbentech',
+  },
+  {
+    id: 4,
+    pName: 'BENtech',
+    pAuthure: 'Benjamin &nbsp; &bull; &nbsp; Back End Dev &nbsp;&bullet;&nbsp; 2022',
+    pPic: 'photo/ben/ben4.png',
+    pText: 'An open source code, created and designed by Benjamin Ejike while studying with microverse, kindly report any kink of copyright.',
+    pTechs: ['Html', 'Css', 'JavaScript'],
+    pLive: 'https://osbentech.github.io/Portfolio/',
+    pSource: 'https://github.com/osbentech',
+  },
+];
+function popFill(arr, num) {
+  const proj = arr[num];
+  popProj.textContent = proj.pName;
+  popName.innerHTML = proj.pAuthure;
+  popPic.src = proj.pPic;
+  popTex.textContent = proj.pText;
+  popUpLive.attributes.href.nodeValue = proj.popUpLive;
+  popUpSource.attributes.href.nodeValue = proj.pSource;
+  popList.innerHTML = '';
+  proj.pTechs.forEach((item) => {
+    const pl = document.createElement('li');
+    pl.innerHTML = item;
+    pl.classList.add('pplist');
+    popList.append(pl);
+  });
+}
+popProject.addEventListener('click', () => {
+  popUp.classList.toggle('active');
+  popFill(projArr, 0);
+});
+popProject1.addEventListener('click', () => {
+  popUp.classList.toggle('active');
+  popFill(projArr, 1);
+});
+popProject2.addEventListener('click', () => {
+  popUp.classList.toggle('active');
+  popFill(projArr, 2);
+});
+popProject3.addEventListener('click', () => {
+  popUp.classList.toggle('active');
+  popFill(projArr, 3);
+});
+
+popClose.addEventListener('click', () => {
+  popUp.classList.remove('active');
```

Comment:
- [ ] [OPTIONAL] For best practices and to not repeat your self kindly use a loop only one time to create this data